services/web-app/pages/assets/index/index.module.scss

```
@@ -111,7 +171,7 @@
   margin-top: spacing.$spacing-08;
   @include type.type-style('fluid-heading-03');
   @include breakpoint.breakpoint(md) {
-    margin-top: spacing.$spacing-08 + spacing.$spacing-02;
+    margin-top: spacing.$spacing-08 + spacing.$spacing-06 + spacing.$spacing-02;
```

```suggestion
    margin-top: spacing.$spacing-10 + spacing.$spacing-02;
```
spacing.$spacing-08 + spacing.$spacing-06 = spacing.$spacing-10 = 64px